Submitting job: /common/home/users/d/dh.huang.2023/code/cs701/validate.sh
Current Directory:
/common/home/users/d/dh.huang.2023/code/cs701
Sun Oct 22 21:23:10 2023       
+---------------------------------------------------------------------------------------+
| NVIDIA-SMI 535.104.05             Driver Version: 535.104.05   CUDA Version: 12.2     |
|-----------------------------------------+----------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |         Memory-Usage | GPU-Util  Compute M. |
|                                         |                      |               MIG M. |
|=========================================+======================+======================|
|   0  NVIDIA L40                     On  | 00000000:01:00.0 Off |                    0 |
| N/A   33C    P8              35W / 300W |      4MiB / 46068MiB |      0%      Default |
|                                         |                      |                  N/A |
+-----------------------------------------+----------------------+----------------------+
                                                                                         
+---------------------------------------------------------------------------------------+
| Processes:                                                                            |
|  GPU   GI   CI        PID   Type   Process name                            GPU Memory |
|        ID   ID                                                             Usage      |
|=======================================================================================|
|  No running processes found                                                           |
+---------------------------------------------------------------------------------------+
Linux lexicon 4.18.0-477.21.1.el8_8.x86_64 #1 SMP Tue Aug 8 21:30:09 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
NAME="Rocky Linux"
VERSION="8.8 (Green Obsidian)"
ID="rocky"
ID_LIKE="rhel centos fedora"
VERSION_ID="8.8"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Rocky Linux 8.8 (Green Obsidian)"
ANSI_COLOR="0;32"
LOGO="fedora-logo-icon"
CPE_NAME="cpe:/o:rocky:rocky:8:GA"
HOME_URL="https://rockylinux.org/"
BUG_REPORT_URL="https://bugs.rockylinux.org/"
SUPPORT_END="2029-05-31"
ROCKY_SUPPORT_PRODUCT="Rocky-Linux-8"
ROCKY_SUPPORT_PRODUCT_VERSION="8.8"
REDHAT_SUPPORT_PRODUCT="Rocky Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="8.8"
Architecture:        x86_64
CPU op-mode(s):      32-bit, 64-bit
Byte Order:          Little Endian
CPU(s):              128
On-line CPU(s) list: 0-127
Thread(s) per core:  2
Core(s) per socket:  64
Socket(s):           1
NUMA node(s):        1
Vendor ID:           AuthenticAMD
CPU family:          25
Model:               1
Model name:          AMD EPYC 7763 64-Core Processor
Stepping:            1
CPU MHz:             2450.000
CPU max MHz:         3529.0520
CPU min MHz:         1500.0000
BogoMIPS:            4890.70
Virtualization:      AMD-V
L1d cache:           32K
L1i cache:           32K
L2 cache:            512K
L3 cache:            32768K
NUMA node0 CPU(s):   0-127
Flags:               fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx mmxext fxsr_opt pdpe1gb rdtscp lm constant_tsc rep_good nopl nonstop_tsc cpuid extd_apicid aperfmperf pni pclmulqdq monitor ssse3 fma cx16 pcid sse4_1 sse4_2 movbe popcnt aes xsave avx f16c rdrand lahf_lm cmp_legacy svm extapic cr8_legacy abm sse4a misalignsse 3dnowprefetch osvw ibs skinit wdt tce topoext perfctr_core perfctr_nb bpext perfctr_llc mwaitx cpb cat_l3 cdp_l3 invpcid_single hw_pstate ssbd mba ibrs ibpb stibp vmmcall fsgsbase bmi1 avx2 smep bmi2 erms invpcid cqm rdt_a rdseed adx smap clflushopt clwb sha_ni xsaveopt xsavec xgetbv1 xsaves cqm_llc cqm_occup_llc cqm_mbm_total cqm_mbm_local clzero irperf xsaveerptr wbnoinvd amd_ppin brs arat npt lbrv svm_lock nrip_save tsc_scale vmcb_clean flushbyasid decodeassists pausefilter pfthreshold v_vmsave_vmload vgif v_spec_ctrl umip pku ospke vaes vpclmulqdq rdpid overflow_recov succor smca fsrm
MemTotal:       527671936 kB
Running python validate.py --batch=256 --phase=1 --model=1
device:  cuda
model:  1 
classes:  10 
checkpoint:  0 
batch:  256 
phase:  1
loaded checkpoint: /common2/dh.huang.2023/code/cs701/data/checkpoints_phase_1/checkpoint-017.pth
accuracy:  0.8629089301503095
886 results saved to: /common2/dh.huang.2023/code/cs701/results/result_1.txt
phase: 1 - final accuracy:0.895
Validation is completed in 0.0 hours, 0.0 minutes, 8.139 seconds
Running python validate.py --batch=256 --phase=2 --model=1
device:  cuda
model:  1 
classes:  20 
checkpoint:  0 
batch:  256 
phase:  2
loaded checkpoint: /common2/dh.huang.2023/code/cs701/data/checkpoints_phase_2/checkpoint-003.pth
accuracy:  0.6090187505519313
886 results saved to: /common2/dh.huang.2023/code/cs701/results/result_2.txt
phase: 2 - final accuracy:0.540
Validation is completed in 0.0 hours, 0.0 minutes, 9.645 seconds
Running python validate.py --batch=256 --phase=3 --model=1
device:  cuda
model:  1 
classes:  30 
checkpoint:  0 
batch:  256 
phase:  3
loaded checkpoint: /common2/dh.huang.2023/code/cs701/data/checkpoints_phase_3/checkpoint-014.pth
accuracy:  0.374489013096608
886 results saved to: /common2/dh.huang.2023/code/cs701/results/result_3.txt
phase: 3 - final accuracy:0.342
Validation is completed in 0.0 hours, 0.0 minutes, 10.073 seconds
Running python validate.py --batch=256 --phase=4 --model=1
device:  cuda
model:  1 
classes:  40 
checkpoint:  0 
batch:  256 
phase:  4
Traceback (most recent call last):
  File "/common2/dh.huang.2023/code/cs701/validate.py", line 84, in <module>
    checkpoint_load(model, SAVE_PATH, checkpoint)
  File "/common2/dh.huang.2023/code/cs701/utils.py", line 239, in checkpoint_load
    model.load_state_dict(torch.load(f))
                          ^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 791, in load
    with _open_file_like(f, 'rb') as opened_file:
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 271, in _open_file_like
    return _open_file(name_or_buffer, mode)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 252, in __init__
    super().__init__(open(name, mode))
                     ^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/common2/dh.huang.2023/code/cs701/data/checkpoints_phase_4/checkpoint-007.pth'
Running python validate.py --batch=256 --phase=5 --model=1
device:  cuda
model:  1 
classes:  50 
checkpoint:  0 
batch:  256 
phase:  5
Traceback (most recent call last):
  File "/common2/dh.huang.2023/code/cs701/validate.py", line 84, in <module>
    checkpoint_load(model, SAVE_PATH, checkpoint)
  File "/common2/dh.huang.2023/code/cs701/utils.py", line 239, in checkpoint_load
    model.load_state_dict(torch.load(f))
                          ^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 791, in load
    with _open_file_like(f, 'rb') as opened_file:
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 271, in _open_file_like
    return _open_file(name_or_buffer, mode)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 252, in __init__
    super().__init__(open(name, mode))
                     ^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/common2/dh.huang.2023/code/cs701/data/checkpoints_phase_5/checkpoint-030.pth'
Running python validate.py --batch=256 --phase=6 --model=1
device:  cuda
model:  1 
classes:  60 
checkpoint:  0 
batch:  256 
phase:  6
Traceback (most recent call last):
  File "/common2/dh.huang.2023/code/cs701/validate.py", line 84, in <module>
    checkpoint_load(model, SAVE_PATH, checkpoint)
  File "/common2/dh.huang.2023/code/cs701/utils.py", line 239, in checkpoint_load
    model.load_state_dict(torch.load(f))
                          ^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 791, in load
    with _open_file_like(f, 'rb') as opened_file:
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 271, in _open_file_like
    return _open_file(name_or_buffer, mode)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 252, in __init__
    super().__init__(open(name, mode))
                     ^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/common2/dh.huang.2023/code/cs701/data/checkpoints_phase_6/checkpoint-015.pth'
Running python validate.py --batch=256 --phase=7 --model=1
device:  cuda
model:  1 
classes:  70 
checkpoint:  0 
batch:  256 
phase:  7
Traceback (most recent call last):
  File "/common2/dh.huang.2023/code/cs701/validate.py", line 84, in <module>
    checkpoint_load(model, SAVE_PATH, checkpoint)
  File "/common2/dh.huang.2023/code/cs701/utils.py", line 239, in checkpoint_load
    model.load_state_dict(torch.load(f))
                          ^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 791, in load
    with _open_file_like(f, 'rb') as opened_file:
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 271, in _open_file_like
    return _open_file(name_or_buffer, mode)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 252, in __init__
    super().__init__(open(name, mode))
                     ^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/common2/dh.huang.2023/code/cs701/data/checkpoints_phase_7/checkpoint-016.pth'
Running python validate.py --batch=256 --phase=8 --model=1
device:  cuda
model:  1 
classes:  80 
checkpoint:  0 
batch:  256 
phase:  8
Traceback (most recent call last):
  File "/common2/dh.huang.2023/code/cs701/validate.py", line 84, in <module>
    checkpoint_load(model, SAVE_PATH, checkpoint)
  File "/common2/dh.huang.2023/code/cs701/utils.py", line 239, in checkpoint_load
    model.load_state_dict(torch.load(f))
                          ^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 791, in load
    with _open_file_like(f, 'rb') as opened_file:
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 271, in _open_file_like
    return _open_file(name_or_buffer, mode)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 252, in __init__
    super().__init__(open(name, mode))
                     ^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/common2/dh.huang.2023/code/cs701/data/checkpoints_phase_8/checkpoint-013.pth'
Running python validate.py --batch=256 --phase=9 --model=1
device:  cuda
model:  1 
classes:  90 
checkpoint:  0 
batch:  256 
phase:  9
Traceback (most recent call last):
  File "/common2/dh.huang.2023/code/cs701/validate.py", line 84, in <module>
    checkpoint_load(model, SAVE_PATH, checkpoint)
  File "/common2/dh.huang.2023/code/cs701/utils.py", line 239, in checkpoint_load
    model.load_state_dict(torch.load(f))
                          ^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 791, in load
    with _open_file_like(f, 'rb') as opened_file:
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 271, in _open_file_like
    return _open_file(name_or_buffer, mode)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 252, in __init__
    super().__init__(open(name, mode))
                     ^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/common2/dh.huang.2023/code/cs701/data/checkpoints_phase_9/checkpoint-002.pth'
Running python validate.py --batch=256 --phase=10 --model=1
device:  cuda
model:  1 
classes:  100 
checkpoint:  0 
batch:  256 
phase:  10
Traceback (most recent call last):
  File "/common2/dh.huang.2023/code/cs701/validate.py", line 84, in <module>
    checkpoint_load(model, SAVE_PATH, checkpoint)
  File "/common2/dh.huang.2023/code/cs701/utils.py", line 239, in checkpoint_load
    model.load_state_dict(torch.load(f))
                          ^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 791, in load
    with _open_file_like(f, 'rb') as opened_file:
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 271, in _open_file_like
    return _open_file(name_or_buffer, mode)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/common/home/users/d/dh.huang.2023/.conda/envs/chatpdf/lib/python3.11/site-packages/torch/serialization.py", line 252, in __init__
    super().__init__(open(name, mode))
                     ^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/common2/dh.huang.2023/code/cs701/data/checkpoints_phase_10/checkpoint-014.pth'
Job ID: 30576
Cluster: crimson
User/Group: dh.huang.2023/dh.huang.2023
State: COMPLETED (exit code 0)
Nodes: 1
Cores per node: 10
CPU Utilized: 00:04:31
CPU Efficiency: 48.39% of 00:09:20 core-walltime
Job Wall-clock time: 00:00:56
Memory Utilized: 936.01 MB
Memory Efficiency: 1.43% of 64.00 GB
